sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.cs

```diff
@@ -441,6 +432,15 @@ public void FailedToReadEnvironmentVariables(Exception ex)
         [Event(42, Message = "Failed to read environment variables due to an exception. This may prevent the Exporter from initializing. {0}", Level = EventLevel.Warning)]
         public void FailedToReadEnvironmentVariables(string errorMessage) => WriteEvent(42, errorMessage);
 
+        [NonEvent]
+        public void ErrorAddingActivityTagsAsCustomProperties(Exception ex)
+        {
+            if (IsEnabled(EventLevel.Warning))
+            {
+                ErrorAddingActivityTagsAsCustomProperties(ex.FlattenException().ToInvariantString());
+            }
+        }
+
```

Comment:
Moved the NonEvent method to be adjacent to the Event method to improve readability.
This is the pattern used by all other NonEvent/Event methods.